lib/Fmt_ast.ml

```diff
@@ -3271,15 +3295,28 @@ and fmt_value_description ?ext c ctx vd =
     $ fmt_item_attributes c ~pre:(Break (1, 2)) atrs
     $ doc_after )
 
+and fmt_tydcl_param c ctx ty =
+  fmt_core_type ~tydecl_param:true c (sub_typ ~ctx ty)
+  $
+  match ty.ptyp_attributes with
+  | [] | _ :: _ :: _ -> noop
+  | [attr] -> fmt_if_k (is_layout attr) (fmt "@ :@ " $ str attr.attr_name.txt)
```

Comment:
Is it possible to have multiple attributes on a type param? Do any of the others in the compiler apply here?

Reply:
Right - I meant to add a comment about this, which is accurate to the parser but I understand why it looks surprising.  I've done so now; see if it makes sense to you.  I also made a jira ticket in v1.5 to track revisiting the parser decision.